packages/camera/camera_web/lib/src/camera_service.dart

```diff
@@ -307,6 +307,43 @@ class CameraService {
     return const Size(320, 240);
   }
 
+  static const int _kiloBits = 1000;
+  static const int _megaBits = _kiloBits * _kiloBits;
+
+  /// Maps the given [resolutionPreset] to video bitrate.
+  int mapResolutionPresetToVideoBitrate(ResolutionPreset resolutionPreset) {
+    switch (resolutionPreset) {
+      case ResolutionPreset.max:
+      case ResolutionPreset.ultraHigh:
+        return 8 * _megaBits;
+      case ResolutionPreset.veryHigh:
+        return 4 * _megaBits;
+      case ResolutionPreset.high:
+        return 1 * _megaBits;
+      case ResolutionPreset.medium:
+        return 400 * _kiloBits;
+      case ResolutionPreset.low:
+        return 200 * _kiloBits;
+    }
+  }
+
+  /// Maps the given [resolutionPreset] to audio bitrate.
+  int mapResolutionPresetToAudioBitrate(ResolutionPreset resolutionPreset) {
+    switch (resolutionPreset) {
```

Comment:
Should these switches be not exhaustive (or have a `default`/wildcard `_` entry), so more `ResolutionPreset`s can be added later without it being a breaking change? /cc @stuartmorgan (this enum is defined in `platform_interface` IIRC)

`mapResolutionPresetToVideoBitrate` has the same issue.

**(Please, see the comment in the `mapResolutionPresetToSize` right above!)**

Reply:
implemented like `mapResolutionPresetToSize`

Reply:
https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#enum-handling

Reply:
@stuartmorgan implemented using fallback and `ignore: dead_code`